Application/Features/VacationFeature/Commands/UpdateVacationCommand.cs

```diff
@@ -72,7 +66,12 @@ public async Task<UpdateVacationCommandResponse> Handle(UpdateVacationCommandReq
         {
             
             var vacation = await _context.Vacations.FirstOrDefaultAsync(vacation => vacation.Id == request.Id, cancellationToken);
-            
+            if (vacation == null)
+            {
+                throw new NotFoundException(nameof(vacation), request.Id);
+            }
+            //EnsureTheUserHasNotVacationInThisTime(request.UserId, request.VacationStart, request.VacationEnd);
+
```

Comment:
There would be the case like this : let's say I have a vacation for 1.03.2022 to 5.03.2022 and I want to update this vacation for 1.03.2022 to 9.03.2022 but the problem is if I have another vacation for 8.03.2022 to 11.03.2022 and I want to update my vacation to 1.03.2022 to 9.03.2022. Then What should I do should I update this vacation or not ?

Reply:
no, from my point of view it's better to return message that it crosses with another vacation (and maybe dates) so user will be aware

Reply:
Done